lib/lotus/model/adapters/memory/query.rb

```diff
@@ -541,6 +541,19 @@ def negate!
             raise NotImplementedError
           end
 
+          # This method is defined in order to make the interface of
+          # `Memory::Query` identical to `Sql::Query`, but this feature is NOT
+          # implemented
+          #
+          # @raise [NotImplementedError]
+          #
+          # @since x.x.x
+          #
+          # @see Lotus::Model::Adapters::Sql::Query#group!
+          def group
+            raise NotImplementedError
```

Comment:
@jodosha what do you think to add group by for memory adapter? ruby brings us that method.